### PR TITLE
Nesting the result and the output type into ValidatingProperty.

### PR DESCRIPTION
**Pull request by andersio**
#### Checklist
- [x] Updated CHANGELOG.md.

### CODE REVIEW

**Comment by andersio** [comment]
Note that `ValidatorOutput` is renamed to be `Decision`.

File: Sources/ValidatingProperty.swift [hunk 1]
```diff
@@ -254,74 +254,74 @@ public final class ValidatingProperty<Value, ValidationError: Swift.Error>: Muta
...
-}
 
-/// Represents a decision of a validator of a validating property made on a
-/// proposed value.
-public enum ValidatorOutput<Value, Error: Swift.Error> {
-	/// The proposed value is valid.
-	case valid
+	/// Represents a decision of a validator of a validating property made on a
+	/// proposed value.
+	public enum Decision {
```
Comment (sharplet) [inline Sources/ValidatingProperty.swift]: How would you feel about `Status`?

Comment (andersio) [inline Sources/ValidatingProperty.swift]: It is a result of an aggressive action, while status is passive state of a matter.

**Comment by mdiep** [comment]
I don't love `Decision`, but no better options come to mind.

**Pull request merged**